### ISSUE TITLE
fix import of checkstyle config file containing multiple instances/definitions of the same rules

### ISSUE THREAD

**Issue by romani** [checkstyle/sonar-checkstyle#3]
https://github.com/SonarQubeCommunity/sonar-checkstyle/issues/28

Error when creating a new quality profile by importing a checkstyle configuration:

```
NativeException (javax.persistence.NonUniqueResultException: result returns more than one elements):
    org/hibernate/ejb/QueryImpl.java:109:in `getSingleResult'
    org/sonar/jpa/session/JpaDatabaseSession.java:154:in `getSingleResult'
    org/sonar/core/components/DefaultRuleFinder.java:60:in `find'
    org/sonar/plugins/checkstyle/CheckstyleProfileImporter.java:124:in `processRule'
    org/sonar/plugins/checkstyle/CheckstyleProfileImporter.java:97:in `processModule'
    org/sonar/plugins/checkstyle/CheckstyleProfileImporter.java:69:in `importProfile'
    org/sonar/server/rules/ProfilesConsole.java:140:in `importProfile'
    org/sonar/server/ui/JRubyFacade.java:185:in `importProfile'
    /app/controllers/profiles_controller.rb:71:in `create'
    /app/controllers/profiles_controller.rb:69:in `each_pair'
    /app/controllers/profiles_controller.rb:69:in `create'
    /gems/gems/actionpack-2.2.2/lib/action_controller/base.rb:1253:in `perform_action'
[...]
```

Maybe the issue comes from a rule with cardinality set to multiple... but not sure.

File chunk from google_checks.xml from checkstyle repo:

```
...
        <module name="RightCurly"/>
        <module name="RightCurly">
            <property name="option" value="alone"/>
            <property name="tokens" value="CLASS_DEF, METHOD_DEF, CTOR_DEF, LITERAL_FOR, LITERAL_WHILE, LITERAL_DO, STATIC_INIT, INSTANCE_INIT"/>
        </module>
...
```

**Comment by romani** [comment]
Example of how import is failing on import of google_checks.xml 
![sonar-checkstyle-googe-profile-error](https://cloud.githubusercontent.com/assets/812984/21275835/6f41c866-c383-11e6-9c86-44f01a5df4cd.png)

**Comment by romani** [comment]
will be fixed in Checkstyle 7.4, need to wait for release....

**Comment by romani** [comment]
User can update imported config to have "id for each duplicated module. fixed google_checks.xml will be released in checkstyle 7.4
